Foundation/NSJSONSerialization.swift

```diff
@@ -184,8 +184,20 @@ open class JSONSerialization : NSObject {
     
     /* Create a JSON object from JSON data stream. The stream should be opened and configured. All other behavior of this method is the same as the JSONObjectWithData:options:error: method.
      */
-    open class func jsonObject(with stream: InputStream, options opt: ReadingOptions = []) throws -> AnyObject {
-        NSUnimplemented()
+    open class func jsonObject(with stream: InputStream, options opt: ReadingOptions = []) throws -> Any {
+        var buffer = [UInt8](repeating: 0, count: 1024)
+        var data = Data()
+        var bytesRead: Int?
+        repeat {
+            bytesRead = 0
+            if stream.hasBytesAvailable {
+                bytesRead = stream.read(&buffer, maxLength: buffer.count)
```

Comment:
if the bytesRead is less than zero it probably should throw an error derived from the streamError